source/hardware/controllers/hercules_djcontrol_inpulse_300.rst

```diff
@@ -54,58 +54,120 @@ Controls not included in this mapping
 -  Beatmatch guide (Hardware control)
 -  PADS: Slicer/Slicer Loop
 -  PADS: Toneplay
--  PADS: FX
```

Comment:
Please move this "not working" list to a new "Known Issues" section at the bottom of the page.

Reply:
Moved and created a new section: 
cd53b3e